app/controllers/equipment_items_controller.rb

```diff
@@ -0,0 +1,11 @@
+class EquipmentItemsController < ApplicationController
+
+  before_action :authenticate_customer!
+  
+  def show
+    current_customer.populate_attributes
```

Comment:
Is there any way to now call this in every place where we need to fill `current_customer` with data? Can we automate it somehow?